source/_remoteClient/menu.py

```diff
@@ -27,20 +27,20 @@ def __init__(self, client: "RemoteClient") -> None:
 		self._switchToConnectItem()
 		self.muteItem: wx.MenuItem = self.Append(
 			wx.ID_ANY,
-			# Translators: Menu item in NvDA Remote submenu to mute speech and sounds from the remote computer.
+			# Translators: Menu item in Remote Access submenu to mute speech and sounds from the remote computer.
 			_("Mute remote"),
-			# Translators: Tooltip for the Mute Remote menu item in the NVDA Remote submenu.
+			# Translators: Tooltip for the Mute Remote menu item in the Remote Access submenu.
 			pgettext("remote", "Mute speech and sounds from the remote computer."),
 			kind=wx.ITEM_CHECK,
 		)
 		self.muteItem.Enable(False)
 		sysTrayIcon.Bind(wx.EVT_MENU, self.onMuteItem, self.muteItem)
 		self.pushClipboardItem: wx.MenuItem = self.Append(
 			wx.ID_ANY,
-			# Translators: Menu item in NVDA Remote submenu to push clipboard content to the remote computer.
-			_("&Push clipboard"),
-			# Translators: Tooltip for the Push Clipboard menu item in the NVDA Remote submenu.
-			pgettext("remote", "Push the clipboard to the remote computer."),
+			# Translators: Menu item in the Remote Access submenu to send the contents of the clipboard to the remote computer.
+			_("&Send clipboard"),
+			# Translators: Tooltip for the Send clipboard menu item in the Remote Access submenu.
+			pgettext("remote", "Send the text on the clipboard to the remote computer."),
```

Comment:
"on" seems wrong here

```suggestion
			pgettext("remote", "Send the clipboard text to the remote computer."),
```